sbt/plan.sh

```diff
@@ -0,0 +1,36 @@
+pkg_origin=afiore
+pkg_name=sbt
+pkg_version=1.1.6
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="A build tool for Scala, Java, and more"
+pkg_upstream_url="https://www.scala-sbt.org"
+pkg_license=("BSD-3-Clause")
+pkg_source=https://github.com/sbt/sbt/releases/download/v${pkg_version}/sbt-${pkg_version}.tgz
+pkg_shasum=f545b530884e3abbca026df08df33d5a15892e6d98da5b8c2297413d1c7b68c1
+pkg_dirname="$pkg_name-$pkg_version"
+pkg_interpreters=(bin/bash)
```

Comment:
Is there any specific reason for this?

Reply:
sorry, I misread the docs. Build works just fine without it...